Header header
string[] soc_items
float32[] soc_values
